tests/contrib/structlog/test_structlog_patch.py

```diff
@@ -0,0 +1,35 @@
+# This test script was automatically generated by the contrib-patch-tests.py
+# script. If you want to make changes to it, you should make sure that you have
+# removed the ``_generated`` suffix from the file name, to prevent the content
+# from being overwritten by future re-generations.
+
+from ddtrace.contrib.structlog import get_version
+from ddtrace.contrib.structlog.patch import patch
+
+
+try:
+    from ddtrace.contrib.structlog.patch import unpatch
+except ImportError:
+    unpatch = None
+from tests.contrib.patch import PatchTestCase
+
+
+class TestStructlogPatch(PatchTestCase.Base):
+    __integration_name__ = "structlog"
+    __module_name__ = "structlog"
+    __patch_func__ = patch
+    __unpatch_func__ = unpatch
+
+    def assert_module_patched(self, structlog):
+        self.assert_wrapped(structlog.configure)
+
+    def assert_not_module_patched(self, structlog):
+        self.assert_not_wrapped(structlog.configure)
+
+    def assert_not_module_double_patched(self, structlog):
+        self.assert_not_double_wrapped(structlog.configure)
+
+    def assert_module_implements_get_version(self):
```

Comment:
This might conflict with @wconti27 changes

Reply:
@wconti27 merge after me